keyboards/keycapsss/kimiko/kimiko.h

```diff
@@ -0,0 +1,26 @@
+/* Copyright 2020 @ben_roe (keycapsss.com)
```

Comment:
Please delete this file.

Reply:
If i delete this file, i get an error on compile.
```bash
❯ qmk compile -kb keycapsss/kimiko/rev1 -km default 
Ψ Compiling keymap with gmake --jobs=1 keycapsss/kimiko/rev1:default


QMK Firmware 0.21.6
Making keycapsss/kimiko/rev1 with keymap default

avr-gcc (Homebrew AVR GCC 8.5.0_1) 8.5.0
Copyright (C) 2018 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Size before:
   text    data     bss     dec     hex filename
      0   24700       0   24700    607c keycapsss_kimiko_rev1_default.hex

Compiling: .build/obj_keycapsss_kimiko_rev1/src/default_keyboard.c                                 In file included from .build/obj_keycapsss_kimiko_rev1/src/default_keyboard.c:26:
./.build/obj_keycapsss_kimiko_rev1/src/default_keyboard.h:43:10: fatal error: kimiko.h: No such file or directory
 #include "kimiko.h"
          ^~~~~~~~~~
compilation terminated.
 [ERRORS]
 | 
 | 
 | 
gmake[1]: *** [builddefs/common_rules.mk:361: .build/obj_keycapsss_kimiko_rev1_default/.build/obj_keycapsss_kimiko_rev1/src/default_keyboard.o] Error 1
gmake: *** [Makefile:392: keycapsss/kimiko/rev1:default] Error 1
Make finished with errors
```

Reply:
`qmk clean` was the solution. thx removed it.